specification/dartLangSpec.tex

```diff
@@ -15785,7 +15782,7 @@ \subsection{Yield-Each}
 It is a compile-time error if $T_e$ may not be assigned to $T_f$.
```

Comment:
$T_e$ has not been defined. Do you mean $U$?

Reply:
$T_e$ is defined in lines 15773-15779, and this is line 15785.

Reply:
Those lines define $T_f$, not $T_e$.

Reply:
(Unless GitHub is choosing to not show me the correct lines of the correct file. Which it is).